ods-documentation fails as latest is used multiple times

https://github.com/opendevstack/ods-jenkins-shared-library/blob/befbc61c64eab4f309aae0a924ade64d05298fc9/docs/modules/jenkins-shared-library/pages/index.adoc?plain=1#L2

Merge/release of v4.3.0 breaks ods-documentation as LATEST is set in master as well as 4.x branch.

Actually latest is set by antora itself for all pages, so the latest tag could be removed from both page aliases.

```
@antora/cli: 3.1.4
@antora/site-generator: 3.1.4
[16:40:22.246] FATAL (antora): Duplicate alias: latest@ods-jenkins-shared-library::index.adoc (specified as: latest@ods-jenkins-shared-library:ROOT:index.adoc)
    source: docs/modules/jenkins-shared-library/pages/index.adoc in https://github.com/opendevstack/ods-jenkins-shared-library.git (branch: master | start path: docs)
Add the --stacktrace option to see the cause of the error.
Error: Process completed with exit code 1.

```
@braisvq1996

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Changed
+- Remove latest tag in docs ([#1015](https://github.com/opendevstack/ods-jenkins-shared-library/issues/1015))
+
 ## [4.3.0] - 2023-07-03
 ### Added
 - Make IS_GXP property available for CFTP documents ([#996](https://github.com/opendevstack/ods-jenkins-shared-library/issues/996))

--- a/docs/modules/jenkins-shared-library/pages/index.adoc
+++ b/docs/modules/jenkins-shared-library/pages/index.adoc
@@ -1,5 +1,4 @@
 = Jenkins Shared Library
-:page-aliases: latest@ods-jenkins-shared-library:ROOT:index.adoc
 
 OpenDevStack provides a https://jenkins.io/doc/book/pipeline/shared-libraries/[Jenkins Shared Library] which offers three pipelines:
 

